README.md

```diff
@@ -239,6 +239,114 @@ Add the modules that you need as Maven dependencies.
 - **`no.nav.security.jwt.issuer.[issuer shortname].cookiename`** - The value of the cookie containing a ID token (not required, only necessary if your api receives calls directly from a browser)
 - **`no.nav.security.jwt.issuer.[issuer shortname].validation.optional_claims`** - A comma separated list of optional claims that will be excluded from default claims.
 
+### token-client-core
+Module provides a core token client support with supported grants: `jwt_bearer`, `client_credentials`, `token_exchange`, through which applications can gain an access token. 
```

Comment:
Funky use of commas. Something like "The module provides a core token client support with supported grants: `jwt_bearer`, `client_credentials` and `token_exchange` through which applications can gain an access token." ?